.github/actions/install-briefcase/action.yml

```diff
@@ -136,8 +136,8 @@ runs:
       id: install
       shell: bash
       run: |
-        REPO="${{ steps.briefcase-override.outputs.repo || inputs.briefcase-url }}"
-        REF="${{ steps.briefcase-override.outputs.ref || steps.briefcase-derived.outputs.version }}"
+        REPO="${{ steps.briefcase.outputs.repo }}"
+        REF="${{ steps.briefcase.outputs.ref || steps.briefcase-derived.outputs.version }}"
```

Comment:
Not sure I follow the inconsistency here - why is the default value still needed for REF, but not for REPO?

Reply:
Previously, `steps.briefcase-override.outputs.repo` was already being set to `inputs.briefcase-url` earlier on in the Action if `Briefcase-Repo` wasn't specified in the PR Body; so, falling back here was already superfluous. The same is still true now for `steps.briefcase.outputs.repo`. As such, I just removed the fallback.

The Briefcase ref, however, has the extra step of deriving the ref from context; so, that still needs to be resolved here if the ref isn't being specified in the PR Body or as an input.